### PR TITLE
Mod Adoption - CKAN update

### PR DESCRIPTION
**Pull request by zer0Kerbal**
@HebaruSan hopefully 🙏 I got the correct repo this time. :D

Adopting four mods with @QuickBASIC's permission.
1. [MoneroFlags](https://github.com/zer0Kerbal/MoneroFlags)
2. [ISRUThanksButNoTanks](https://github.com/zer0Kerbal/ISRUThanksButNoTanks)
3. [KerbalInventorySystemNoFun](https://github.com/zer0Kerbal/KerbalInventorySystemNoFun)
4. [UniversalStorageStockResourceFuelCell](https://github.com/zer0Kerbal/UniversalStorageStockResourceFuelCell)

Will be uploading to SpaceDock when it is time. Will upload .version files etc 🔜 .

### CODE REVIEW

**Comment by DasSkelett** [comment]
They all don't have a release yet, CI won't succeed until there's something to test.

**Comment by DasSkelett** [comment]
For future reference, previous maintainer agreed with the adoption of those mods in a forum group PM.

**Comment by QuickBASIC** [comment]
Hello, I'm the original maintainer of these mods. I haven't had a chance to test compatibility with recent versions of KSP and no longer play KSP so likely won't be updating them anytime soon.

Please allow him to make any changes necessary so that these will still be available in CKAN once he begins maintaining them.

**Comment by zer0Kerbal** [comment]
- [x] [ISRU Thanks But No Thanks](https://github.com/zer0Kerbal/ISRUThanksButNoTanks/releases)
- [ ] - [ ] [Universal Storage Stock Resource Fuel Cell](https://github.com/zer0Kerbal/UniversalStorageStockResourceFuelCell/releases)
- [ ] - [ ] [Kerbal Inventory System No Fun](https://github.com/zer0Kerbal/KerbalInventorySystemNoFun/releases)
- [ ] - [ ] [Monero Flags](https://github.com/zer0Kerbal/MoneroFlags/releases)

**Comment by DasSkelett** [comment]
Looks like MoneroFlags is still missing a release

**Comment by DasSkelett** [comment]
Do you want the recommendations listed on the forum threads in the netkans as well?

**Comment by zer0Kerbal** [comment]
does `spec_version: 1` need to be 1.2?

File: NetKAN/ISRUThanksButNoTanks.netkan [hunk 1]
```diff
@@ -7,11 +7,14 @@ x_netkan_epoch: 1
...
 resources:
   homepage: https://forum.kerbalspaceprogram.com/index.php?/topic/204750-*
+tags:
+  - config
+  - resources
 depends:
   - name: ModuleManager
+recommends:
+  - name: KerbalChangelog
+  - name: StockalikeMiningExtension
```
Comment (zer0Kerbal) [inline NetKAN/ISRUThanksButNoTanks.netkan:17]: I believe this should be supports

Comment (DasSkelett) [reply NetKAN/ISRUThanksButNoTanks.netkan:17]: The forum thread mentions them as recommends and supports:
grafik

I guess we can put them in as both if you want. `supports` will make ISRUThanksButNoThanks show up when someone installs KerbalChangelog or StockalikeMiningExtension, `recommends` will make KerbalChangelog or StockalikeMiningExtension show up when someone installs ISRUThanksButNoThanks.

**Comment by DasSkelett** [comment]
> kindly adjust to GameData/QuickBASIC (as that is the way the original author spelled it.

I made them match the zips, which spell it `QuickBasic`. CKAN should reflect manual installs, so if you want it spelled differently, please also change the zips files.

**Comment by zer0Kerbal** [comment]
fixed the missing release. 
missed that(QuickBasic vs QuickBASIC) - sometimes automation is a blessing and a 🥾 🔙 side
will fix in next set of releases 🔜

**Comment by zer0Kerbal** [comment]
actually - fixing now.

**Comment by zer0Kerbal** [comment]
soon just happened. all updated to `GameData/QuickBASIC/`
thank you for all your assistance!

**Comment by DasSkelett** [comment]
Looks good so far. What should we do with the recommendations, add them as supports as well? Or only as supports, or only as recommends?
Also I hope it's okay to leave out the suggestions, they are _a lot_.

**Comment by zer0Kerbal** [comment]
*chuckle* 🤣 just a few. :D 
I'd leave them for now. I can always go back when I do the final release in a couple of weeks and tweak them. 
Thank you 🦃

**Comment by DasSkelett** [comment]
In this case, I think we are good to go. Thank you for the PR!

**Pull request merged**